### ISSUE TITLE
python 3.5 was released

### ISSUE THREAD

**Issue by egolus** [baudren/NoteOrganiser#82]
https://www.python.org/downloads/release/python-350/

we should check if everything works with 3.5 and maybe add it to travis?

**Comment by baudren** [comment]
Yes, that would be nice. Can you go ahead and check it? Thanks in advance,

**Comment by baudren** [comment]
I create a branch to test the support. As it could have been expected, there is not yet a PySide install for py35, so it fails there.

**Comment by egolus** [comment]
PySide can't be installed on python 3.5 yet. There is a [fix](https://github.com/PySide/pyside-setup/commit/e01ce681115a24f6bee0281d0c85f71b098bd8a1) in the `pyside setup` repository, but they didn't release a fix version through pypi yet
